src/components/Text/styles.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/variables.scss";
+
+@mixin HeadingText {
+  color: $color_white;
+  font-family: "Omnes";
+  font-weight: 700;
+  line-height: 125%;
+}
+
+@mixin BodyText {
+  color: $color_white;
+  font-family: "Operator Mono";
+  font-weight: 350;
+  line-height: 125%;
+  margin-top: $spacing_xx_small;
```

Comment:
could we name spacing variables like `$spacing-xs`, `$spacing-sm`, `$spacing-md`, `$spacing-lg`, etc?

Reply:
Sure